### PR TITLE
250 vbo009001 fails on apu

### PR DESCRIPTION
**Pull request by JanPrusinowski**

### CODE REVIEW

File: platform-configs/novacustom-v560tnd.robot [hunk 1]
```diff
@@ -16,11 +16,3 @@ ${CPU_MAX_FREQUENCY}=               4800
 ${CPU_MIN_FREQUENCY}=               200
 
 ${NVIDIA_GRAPHICS_CARD_SUPPORT}=    ${TRUE}
-
-${POWER_CTRL}=                      none
```
Comment (macpijan) [inline platform-configs/novacustom-v560tnd.robot:20]: @JanPrusinowski I wonder why do we need such a change in this MR?

Comment (JanPrusinowski) [reply platform-configs/novacustom-v560tnd.robot]: No we dont. I have cleaned the Commits - rebased to current develop. I noticed that some of my fixes from this PR were merged already with other issues.

File: platform-configs/pcengines-apu4.robot [hunk 2]
```diff
@@ -3,4 +3,5 @@ Resource    include/pcengines.robot
 
 
 *** Variables ***
-${DMIDECODE_PRODUCT_NAME}=      apu4
+${DMIDECODE_PRODUCT_NAME}=          apu4
+${INTERNAL_PROGRAMMER_CHIPNAME}=    W25Q64BV/W25Q64CV/W25Q64FV
```
Comment (krystian-hebel) [inline platform-configs/pcengines-apu4.robot:7]: Is this required only for apu4 or for all apus? IIRC flash is the same on all platforms.

In any case, this should be in separate commit, it has nothing to do with the keyword added in the other file.

**Comment by macpijan** [comment]
It's actually easier to create new MRs rather than fixing this one.

1. https://github.com/Dasharo/open-source-firmware-validation/pull/585/files
